src/executables/unload_schema.c

```diff
@@ -897,6 +845,11 @@ export_serial (extract_context & ctxt, print_output & output_ctx)
 	    }
 	  output_ctx (";\n");
 
+	  if (db_get_int (&values[SERIAL_STARTED]) == 1)
+	    {
+	      output_ctx ("SELECT %s%s%s.NEXT_VALUE;\n ", PRINT_IDENTIFIER (db_get_string (&values[SERIAL_NAME])));
```

Comment:
What about use of '**NEXTVAL**' instead of '**NEXT_VALUE**'

Reply:
I know NEXT_VALUE and NEXTVAL mean the same thing. Is there a reason why you recommend NEXTVAL?
For reference, I used NEXT_VALUE when extracting SERIAL generated by AUTO_INCREMENT.

Reply:
Definitely, 'next_val' could working, but I think 'nextval' is more common term for next value for SERIAL/SEQUENCE. Its your choice.